integration/testrunner/grpc.go

```diff
@@ -119,19 +125,28 @@ func testGRPCEndpoint(ctx context.Context, t *testing.T, g *GRPC) {
 	t.Run("hello_req_unary_success", func(t *testing.T) {
 		t.Parallel()
 		id := uuid.New().String()
+		t.Logf("using uuid %s", id)
 		_, err := g.TalkerClient.Hello(ctx, &talkerpb.HelloRequest{Message: "Some Message", Target: id})
 		if err != nil {
 			t.Errorf("could not greet: %v", err)
 		}
-		query := g.makeQueryForGRPCUnary(id)
-		queryIfAuditLogExistsWithRetries(ctx, t, query, g.Config, "unary_hello")
+		query := g.makeQueryForGRPC(id)
+		t.Log("querying bigquery table using the following statement:")
+		t.Log(query.Q)
```

Comment:
```suggestion
		t.Logf("querying BigQuery:\n%s", query.Q)
```
Similar for other places